Scripts/operations.py

```diff
@@ -602,51 +602,43 @@ def check_duplicates(current, config, miscData, allVideoCommentsDict, videoID):
   # Run the actual duplicate checking
   for authorID, authorCommentsList in allVideoCommentsDict.items():
     # Don't scan channel owner, current user, or any user in whitelist. Also don't bother if author is already in matchedCommentsDict
-    if auth.CURRENTUSER.id == authorID or miscData.channelOwnerID == authorID or authorID in miscData.resources['Whitelist']['WhitelistContents'] or any(authorID == value['authorID'] for key,value in current.matchedCommentsDict.items()):
-      scannedCount +=1
-      print(f" Analyzing For Duplicates: [ {scannedCount/authorCount*100:.2f}% ]   (Can be Disabled & Customized With Config File)".ljust(75, " "), end="\r")
-    else:
-      numDupes = 0
+    if auth.CURRENTUSER.id != authorID and miscData.channelOwnerID != authorID and authorID not in miscData.resources['Whitelist']['WhitelistContents'] and all(authorID != value['authorID'] for key,value in current.matchedCommentsDict.items()):
       commentTextList = []
-      matchedIndexes = []
       for commentDict in authorCommentsList:
         # Adding to use as lower case, because levenshtein is case sensitive. Also, root domain list is ingested as lower case, so necessary to compare
-        commentTextList.append(commentDict['commentText'].lower())
-
-      # Count number of comments that are similar to at least one other comment
-      if len(commentTextList) > 1:
-        for i,x in enumerate(commentTextList):
-          # Check length of comment against minimum, but override if a domain is detected
-          if len(x) >= minimum_duplicate_length or (len(x) >= 6 and any(f".{domain}" in x for domain in domainList)):
+        text = commentDict['commentText'].lower()
+        # Check length of comment against minimum, but override if a domain is detected
+        if len(x) >= minimum_duplicate_length or (len(x) >= 6 and any(f".{domain}" in x for domain in domainList)):
+          commentTextList.append(text)
+
+      # If Levenshtein distance is 1.0, then only check if comment text is exactly the same
+      if levenshtein == 1.0:
+        uniqueMatches = len(commentTextList) - len(set(commentTextList))
+      # If Levenshtein distance is 0, don't check at all, just count number of comments by user
+      elif levenshtein == 0.0:
+        uniqueMatches = len(commentTextList) - 1
+      else:
+        # Count number of comments that are similar to at least one other comment
+        matchedIndexes = []
+        if len(commentTextList) > 1:
+          for i,x in enumerate(commentTextList):
             for j in range(i+1,len(commentTextList)):
               y = commentTextList[j]
-              # If Levenshtein distance is 1.0, then only check if comment text is exactly the same
-              if levenshtein == 1.0 and x == y: 
```

Comment:
The `x == y` should have been placed in an inner check. In the old implementation it would fall back to the fuzzy comparison when the strings are not similar.